articles/server-platforms/rails/00-introduction.md

```diff
@@ -0,0 +1,78 @@
+## Ruby on Rails - Introduction
+This is the very beginning of a simple, practical and multi-step quickstart that will guide you through managing authentication in your Ruby on Rails apps with Auth0.
+
+### Seed & Samples
+
+The Auth0 RoR samples provide a seed project and a set of samples.
+- The seed is a regular RoR app with nothing on Auth0 in it except for the dependencies listed in this article and your Auth0 API keys. It serves as the starting point for building an app that works with Auth0 from scratch.
+- There is a sample for each step on this quickstart. The samples are nothing but the seed plus the contents specified on each step, allowing you to understand how to implement a particular functionality related to Auth0.
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+* Ruby 2.3.1
+* Rails 4.2.6
+:::
+
+<%= include('../../_includes/_package', {
+  pkgRepo: 'omniauth-auth0',
+  pkgBranch: 'master',
+  pkgPath: 'examples/ruby-on-rails-webapp',
+  pkgFilePath: null,
+  pkgType: 'server'
+}) %>
+
+### 1. Create an Application
+
+Get your credentials (Domain, Client ID, and Client Secret) from the dashboard. The sample will be configured with your Default App credentials.
+
+#####TODO: IMAGE
```

Comment:
missing image, you could copy this one https://github.com/auth0/docs/blob/master/media/articles/java/app_dashboard.png